demo-simple/src/main/java/com/novoda/downloadmanager/demo/DownloadBatchStatusView.java

```diff
@@ -0,0 +1,82 @@
+package com.novoda.downloadmanager.demo;
+
+import android.content.Context;
+import android.support.constraint.ConstraintLayout;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.TextView;
+
+import com.novoda.downloadmanager.DownloadBatchStatus;
+
+import static com.novoda.downloadmanager.DownloadBatchStatus.Status.ERROR;
+
+public class DownloadBatchStatusView extends ConstraintLayout {
+
+    private TextView downloadBatchStatusView;
+
+    private DownloadBatchStatusListener downloadBatchStatusListener;
+
+    public DownloadBatchStatusView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+        View.inflate(getContext(), R.layout.merge_download_batch_status, this);
+
+        downloadBatchStatusView = findViewById(R.id.download_batch_status);
+        View pauseDownloadView = findViewById(R.id.button_pause_downloading);
```

Comment:
I'd avoid the local variables because if you create a `NO_OP` implementation for the `DownloadBatchStatusListener` then you can collapse all this code in just two lines

Reply:
979d0f9